api/src/utils/order-utils.js

```diff
@@ -120,6 +120,8 @@ const TAG_MAP = {
   gpu: 9,
 };
 
+const IS_STRICT_FILTER = false;
```

Comment:
not sure we want to store it in a constant, if we do I would rename it `IS_STRICT_FILTER_DEFAULT_VALUE`

Reply:
commit 35d6cf28c5aa5a3517901bcb3f31bf64036c256b